From ec9c2d02311eba6b884d6e441a60d8536cc8dff9 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:43:52 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(a6791e4)..lineage-16.0-android-9.0.0_r31(9f3cdcb) (DO NOT
 SUBMIT)

Bill Yi (2):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (2):
 - Snap for 5044688 from 276f0cc2830af7267aa06114655565a50921b548 to
   pi-qpr2-release
 - Snap for 5073521 from 075621a33ab1380d669646ec40e184ddfa820253 to
   pi-qpr2-release

Change-Id: I56fc4e1459af293948ab498be8750a39c26010cd

Change-Id: I9eb06b23142bd0f77fea7a982348bca3f39ee3b5
---
 res/values-ca/strings.xml     | 2 +-
 res/values-zh-rCN/strings.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index c191343..edc2b10 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -25,6 +25,6 @@
     <string name="local_source_name" msgid="6487253091085518505">"Fotos del dispositiu"</string>
     <string name="need_to_configure" msgid="1269102638159389032">"No s\'ha seleccionat cap foto."</string>
     <string name="photodream_select_all" msgid="6364858563337927827">"Selec. tot"</string>
-    <string name="photodream_select_none" msgid="8816452124894525861">"Anul. sel. tot"</string>
+    <string name="photodream_select_none" msgid="8816452124894525861">"Desselecciona tot"</string>
     <string name="no_photos" msgid="4262083523147564199">"No hi ha cap foto al dispositiu."</string>
 </resources>
diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index 92cd1e6..67af3fd 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -20,7 +20,7 @@
     <string name="table_screensaver_name" msgid="1150228032617397055">"照片桌面"</string>
     <string name="flipper_screensaver_name" msgid="6988572107391523141">"相框"</string>
     <string name="posts_album_name" msgid="5628151551239729515">"信息中的照片"</string>
-    <string name="unknown_album_name" msgid="2135521368192091267">"未命名相册"</string>
+    <string name="unknown_album_name" msgid="2135521368192091267">"未命名影集"</string>
     <string name="stock_photo_album_name" msgid="3150527596227525460">"图库照片"</string>
     <string name="local_source_name" msgid="6487253091085518505">"设备中的照片"</string>
     <string name="need_to_configure" msgid="1269102638159389032">"未选择任何照片。"</string>
-- 
2.17.1

